app/models/user.rb

```diff
@@ -185,4 +167,23 @@ def mobile_number=(new_number)
       create_mobile_phone!(number: new_number)
     end
   end
+
+  # For legacy reasons, the User database table includes fields for image, provider, email, and uid
+  # These should not be used directly, and may be blank for some users, instead use the identity information
+
+  def image_url
+    identity&.image_url
+  end
+
+  def provider
+    identity&.provider
+  end
+
+  def uid
+    identity&.uid
+  end
+
+  def email
+    identity&.email
+  end
```

Comment:
Need to get rid of this, we want to have an email address on the user and it may be different to the one for the profile